dist/index.html

```diff
@@ -17,16 +17,16 @@ <h2>Recent scores</h2>
                  <button id="refresh-btn">Refresh</button>
                 </div>
                 <ul class="list">
-                    <li>Name: 100</li>
+                    <!-- <li>Name: 100</li>
                     <li>Name: 20</li>
                     <li>Name: 50</li>
                     <li>Name: 78</li>
                     <li>Name: 125</li>
                     <li>Name: 77</li>
-                    <li>Name: 42</li>
+                    <li>Name: 42</li> -->
```

Comment:
Kindly make sure you remove all the commented code lines in your code base for best practices and make it cleaner and professional.